src/python_testing/TC_MCORE_FS_1_1.py

```diff
@@ -47,7 +47,7 @@
 
 import chip.clusters as Clusters
 from chip import ChipDeviceCtrl
-from chip.testing.tasks import Subprocess
```

Comment:
in 1.4 branch, the path seems to be chip.testing.tasks for the WHL:

https://github.com/rquidute/connectedhomeip/blob/v1.4-branch/src/python_testing/matter_testing_infrastructure/BUILD.gn#L34